src/main/java/seedu/address/storage/JsonAdaptedNote.java

```diff
@@ -62,6 +66,20 @@ public boolean isClientNote() {
      * @throws IllegalValueException if there were any data constraints violated in the adapted note.
      */
     public Note toModelType() throws IllegalValueException {
+        if (contents == null) {
+            throw new IllegalValueException(
+                    String.format(MISSING_FIELD_MESSAGE_FORMAT, Note.class.getSimpleName()));
+        }
+
+        if (!Note.isValidNote(contents)) {
+            throw new IllegalValueException(Note.MESSAGE_CONSTRAINTS);
+        }
+
+        if (countryCode == null) {
+            throw new IllegalValueException(
+                    String.format(MISSING_FIELD_MESSAGE_FORMAT, Country.class.getSimpleName()));
```

Comment:
Should lowercase the getSimpleName so the displayed message will look less weird

Reply:
done